dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/services/BulkDataRequestEmailBuilder.kt

```diff
@@ -0,0 +1,68 @@
+package org.dataland.datalandcommunitymanager.services
+
+import org.dataland.datalandbackendutils.email.BaseEmailBuilder
+import org.dataland.datalandbackendutils.email.Email
```

Comment:
Could we also generalize the email generation? Basicly e-mails are build the same was, only sender, recipients and content varies

Reply:
That is what I created the BaseEmailBuilder for. But I see, that there is more possible. See comment on BaseEmailBuilder.

Reply:
Is now a libary, so closing this